todo-mvc/src/widgets/createTodoItem.ts

```diff
@@ -17,82 +17,125 @@ interface TodoItemState extends WidgetState, StatefulChildrenState {
 	completed?: boolean;
 }
 
-export interface TodoItemOptions extends WidgetOptions<TodoItemState>, ParentMapMixinOptions<Widget<TodoItemState>>, StatefulChildrenOptions<Child, TodoItemState> { }
+export interface TodoItemOptions extends WidgetOptions<TodoItemState>, StatefulChildrenOptions<Child, TodoItemState> { }
 
-export type TodoItem = Widget<TodoItemState> & ParentMapMixin<Child>;
+export type TodoItem = Widget<TodoItemState>;
 
 export interface TodoItemFactory extends ComposeFactory<TodoItem, TodoItemOptions> { }
 
-function manageChildren() {
-	const todoItem = <TodoItem> this;
-	const label = <Widget<WidgetState>> todoItem.children.get('label');
-	const checkbox = <Widget<WidgetState>> todoItem.children.get('checkbox');
-	const editInput = <Widget<WidgetState>> todoItem.children.get('editInput');
+interface TodoItemChildrenItem {
```

Comment:
> Feels like anywhere that uses the `createChildren` API would be require a similar interface to `TodoItemChildrenItem`

I need to look at it again, because the map returned from `createChildren` might very well have this shape... I do think it is a legacy from my iteration on things.  Apologies.

> and even the concept of the `childrenMap` could probably be abstracted away (I think)

I thought about that too when doing this... the challenge is that the current `createParentMapMixin` makes assumptions that map `key === Child.id`, but I see that as something we should iterate on versus holding up landing this.